src/EditorFeatures/CSharpTest/Completion/CompletionProviders/OverrideCompletionProviderTests.cs

```diff
@@ -777,6 +777,23 @@ public class SomeClass : Base
             await VerifyItemExistsAsync(markup, "goo(int x, out string y)", "void Base.goo(int x, out string y)");
         }
 
+        [WpfFact, Trait(Traits.Feature, Traits.Features.Completion)]
+        [WorkItem(24490, "https://github.com/dotnet/roslyn/issues/24490")]
+        public async Task InParameter()
+        {
+            var markup = @"public abstract class Base
+{
+    public abstract void goo(int x, in string y);
```

Comment:
❗️ This needs to be in a different assembly to be a verification test for #24490.

Reply:
Ah, makes sense. Thanks

Reply:
Got a proper repro and updated PR